website/docs/source/v2/provisioning/ansible_intro.html.md

```diff
@@ -156,6 +156,8 @@ Vagrant.configure(2) do |config|
     ansible.groups = {
       "group1" => ["machine1"],
       "group2" => ["machine2"],
+      "group3" => ["machine[01:50]"],
+      "group4" => ["machine-[a:d]"],
```

Comment:
and group4 to illustrate the unsafety aspect, e.g. with `"group4" => ["other_node-[a:d]"],`